index.html

```diff
@@ -102,6 +124,7 @@ <h2 class="bold">Question</h2>
             <path
               d="M21 13v10h-6v-6h-6v6h-6v-10h-3l12-12 12 12h-3zm-1-5.907v-5.093h-3v2.093l3 3z"
             />
+            <a href="##" alt="Home"></a>
```

Comment:
Wouldn't it be better to place the a element around the svg instead of placing it inside?